Bug fixes

Adrian2k opened the pull request:
Resolves #80, resolves #74

nepetadosmil commented on VATSpy.dat:
```diff
@@ -17863,7 +17863,7 @@ LWSS|Skopje||LWSS
 LYBA|Beograd||LYBA
 LZBB|Bratislava||LZBB
 MDCS|Santo Domingo||MDCS
-MHTG|Central America||MHTG
```
Just clarifying something about the position name with them; should be fine to merge after

nepetadosmil replied:
New position's (MHCC_CTR) radio callsign is "Cenamer Centre"

Adrian2k commented:
I think I have renamed it to Cenamer Centre now, but yet to see it online to see it is working. The FIR is called Central America.

Adrian2k commented:
I have renamed Central America FIR to Cenamer FIR now and added callsign Centre. However the FIR is actually called Central America.

Ready for merge now.
